capsules/src/spi.rs

```diff
@@ -35,6 +47,15 @@ pub struct Spi<'a, S: SpiMasterDevice + 'a> {
     kernel_len: Cell<usize>,
 }
 
+pub struct SpiSlave<'a, S: SpiSlaveDevice + 'a> {
+    spi_slave: &'a S,
+    busy: Cell<bool>,
+    app: MapCell<SlaveApp>,
```

Comment:
This should be a `Container` not a `MapCell` since you're going to want to (a) tax this memory to the process as well as (b) have a separate copy for each calling process.